### PR TITLE
Pulling Question Data into it's own module, tieData

### PR DESCRIPTION
**Pull request by theresa-lee**

### CODE REVIEW

**Comment by theresa-lee** [comment]
@seanlip PTAL

File: client/question/question.html [hunk 1]
```diff
@@ -10,9 +10,10 @@
     <script src="../../third_party/codemirror-5.19.0/mode/python/python.js"></script>
     <script src="../../third_party/codemirror-5.19.0/addon/edit/matchbrackets.js"></script>
     <script src="../../third_party/ui-codemirror-0.3.0/ui-codemirror.min.js"></script>
-    <script src="../shared/modules/config.js"></script>
+    <script src="../config/config.js"></script>
```
Comment (seanlip) [inline client/question/question.html]: Do you think we could group these imports by module? I.e. all the ../config stuff first, then one blank line, then all the ../data stuff, then one blank line, then all the other stuff? Slightly easier to grok that way, I think.

**Comment by seanlip** [comment]
@eyurko please also note the change in file structure. We got rid of shared/modules and decided to make all the modules top-level (so client/question, client/menu, client/config, client/data) otherwise it starts to get a bit messy because e.g. the QuestionPage module depends on the data module which depends on the config module.

File: client/question/question.html (hunk 1 above)
Comment (theresa-lee) [inline client/question/question.html]: Done.

**Comment by seanlip** [comment]
Oh, wait, not quite yet. Tests are failing.

**Comment by theresa-lee** [comment]
Okay, sorry about that. @eyurko I restructured the server_URL constant into tieConfig since that seemed like a more appropriate place for that. I think we'll want to move the server handler stuff in there as well but perhaps in a different PR.

**Comment by seanlip** [comment]
NB: as discussed, we might want  to name that file something more general than serverUrl.js. Perhaps deploymentSpecificConfig.js (or similar).

**Pull request merged**